framework/context/resourcecanceledcontext/resource_canceled_context.go

```diff
@@ -1,6 +1,6 @@
-// Package canceledcontext stores and accesses the canceled in
+// Package resourcecanceledcontext stores and accesses the canceled in
 // context.Context.
-package canceledcontext
+package resourcecanceledcontext
```

Comment:
Package `resourcecanceledcontext` is to cancel the resource only. Following resources are still reconciled even though a resource got canceled.